html-css/README.md

```diff
@@ -8,7 +8,13 @@ If you are not familiar with linters, read [root level README](../README.md).
 
 Please do the following **steps in this order**:
 
-1. In the first commit of your feature branch create a `.github/workflows` folder and add a copy of [`.github/workflows/linters.yml`](.github/workflows/linters.yml) to that folder.
+1. In the first commit of your feature branch create a `.github/workflows`
+folder and add a copy of
+[this file](https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.github/workflows/linters_css.yml)
+to that folder; name this file `linters.yml`. **Attention**, if you already
+completed the Sass lessons (CSS preprocessors) then you can use
+[this file](https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.github/workflows/linters_sass.yml)
+instead to lint only your Sass files.
```

Comment:
We will also need to modify the instructions for the local setup for Styelint, as it mimics the process we do on GHA:

https://github.com/microverseinc/linters-config/tree/master/html-css#stylelint

Reply:
We can create a new section at the bottom...